sympy/functions/special/delta_functions.py

```diff
@@ -404,44 +404,46 @@ def _sage_(self):
 
 
 class Heaviside(Function):
-    r"""
-    Heaviside Piecewise function.
+    r"""Heaviside step function.
+
+    .. versionchanged:: 1.9
+        ``Heaviside(0)`` now returns 1/2; previously, the default was that
+        the value at 0 was undefined.
```

Comment:
We don't have anything about this in the style guide but I wouldn't put the versionchanged at the top of the docstring. Probably it should go at the bottom.

Reply:
Will be moved ;)